### PR TITLE
[PROTOCOL-182] ATMGovernance: Increase Code Coverage

### PR DESCRIPTION
**Pull request by sypeer**
It is:
- Increased code coverage and docs

It includes:
- Additional ATMFactory tests
- Docs for ATMFactory implementation and interface

Screen Shot 2020-08-18 at 8 17 57 PM

### CODE REVIEW

**Review by Juan-cc — APPROVED** [review]

File: test/ATM/ATMFactoryInitializeTest.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                settingsAddress = NULL_ADDRESS;
+            }
+            if (atmSettingsInstance == 0) {
+                atmSettingsAddress = atmSettings.address;
+            } else {
+                atmSettingsAddress = NULL_ADDRESS;
+            }
+
+            try {
+                console.log(settingsAddress, atmSettingsAddress);
```
Comment (salazarguille) [inline test/ATM/ATMFactoryInitializeTest.js:42]: Please, remove this console.log

File: test/ATM/ATMFactorySetATMSettingsTest.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+// JS Libraries
```
Comment (salazarguille) [inline test/ATM/ATMFactorySetATMSettingsTest.js:1]: Please, verify the folder is ```test/atm``` (lowercase) for all unit tests.

File: test/ATM/ATMFactoryInitializeTest.js (hunk 1 above)
Comment (sypeer) [reply test/ATM/ATMFactoryInitializeTest.js:42]: Removed

File: test/ATM/ATMFactorySetATMSettingsTest.js (hunk 2 above)
Comment (sypeer) [reply test/ATM/ATMFactorySetATMSettingsTest.js:1]: Changed to lowercase

**Pull request merged**